src/QmlControls/QGCSlider.qml

```diff
@@ -23,6 +23,7 @@ Slider {
     property bool zeroCentered: false
     property bool displayValue: false
     property bool indicatorBarVisible: true
+    property real groundIndicator: NaN
```

Comment:
If I understand correctly, QGCSlider is not only used for altitudes, so "groundIndicator" might not always make sense. Also it's not very obvious that the value of groundIndicator sets how much the slider is filled.

How about "thresholdIndicatorValue" or something like that?

Reply:
agree, changed